Docker-compose-web-srv-https-redirect/SyncFldr/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+
+services:
+    nginx-on-centos:
+        build: centos-nginx/
```

Comment:
it will be better to use also **_image_** parameter to set name of building image